kuka_eki_hw_interface/krl/RobotCommandServer.xml

```diff
@@ -0,0 +1,25 @@
+<ETHERNETKRL>
+   <CONFIGURATION>
+      <EXTERNAL>
+         <TYPE>Client</TYPE>
+      </EXTERNAL>
+      <INTERNAL>
+         <ENVIRONMENT>Program</ENVIRONMENT>
+         <ALIVE Set_Flag="2" />

```

Comment:
The flags are system variables that are (optionally) set/cleared by the EKI interface when connections are made.  I am using them to prevent issues with publishing data when no clients are connected.  Flag 1 is used by the state server, flag 2 by the command server.  I documented this in the KRL code as Kuka's KRL parser does not seem to allow comments... :disappointed:

Reply:
Perhaps the comments situation is not as 'sad' as you made it sound, judging from your later comments ? :)